test/flash.test.js

```diff
@@ -228,6 +225,8 @@ QUnit.test('creates FlashSourceBuffers for video/mp2t', function() {
 QUnit.test('waits for the next tick to append', function() {
   let sourceBuffer = this.mediaSource.addSourceBuffer('video/mp2t');
 
+  sourceBuffer.transmuxer_.postMessage = postMessage_;
```

Comment:
We can't because the sourceBuffer isn't created in the beforeEach since not every test uses the same MIME type